qulice-checkstyle/src/test/resources/com/qulice/checkstyle/ChecksTest/NoJavadocForOverriddenMethodsCheck/Invalid.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * This is not a real Java class. It won't be compiled ever. It is used
+ * only as a text resource in integration.ChecksIT.
+ */
+
+public final class Invalid {
+
+    @Override
+    public void overridden() {
```

Comment:
@mkordas I don't think it is problematic to have a valid case here. But I'll do it like the other one.